src/shippingservice/Dockerfile

```diff
@@ -18,14 +18,16 @@ ENV PROJECT github.com/GoogleCloudPlatform/cloud-ops-sandbox/src/shippingservice
 WORKDIR /go/src/$PROJECT
 
 # restore dependencies
-COPY go.mod go.sum *.go ./
+COPY go.mod go.sum main.go quote.go tracker.go ./
 COPY genproto ./genproto/
 RUN go mod vendor -v
+
+COPY . .
```

Comment:
Does this make lines 21, 22 redundant? Since we end up copying everything anyway. 

Or we can just be explicit about what additional vendor/files we're copying over.

Reply:
ohhh, it was left after some try and error process. I removed the `COPY . .` line.